Added regional association plot

ericreed opened the pull request:
Also minor changes to GWAA.R that will make users able to run GWAS on binary outcomes.

dkulp2 commented on R/globals.R:
```diff
@@ -9,12 +9,11 @@ out.dir <- data.dir                     # may want to write to a separate dir to
-protein.coding.coords.fname <- sprintf("%s/ProCodgene_coords.csv", data.dir)
+clinical.fn <- sprintf("%s/GWAStutorial_clinical.csv", data.dir) 
+onethou.fn <- lapply(c(info='info',ped='ped'), function(n) sprintf("%s/chr16_1000g_CEU.%s", data.dir, n))
+protein.coding.coords.fname <- sprintf("%s/ProCodgene_coords.csv", out.dir)
```
I'm confused.  You changed this back to out.dir after I set it to data.dir, I think.
I thought your previous PR pointed out that protein.coding.coords.fname was an _input_ and therefore located in `data.dir`.  `out.dir` is supposed to be a location possibly different than `data.dir` where results are written.

ericreed commented:
Here I added a code file by split code3.R into two files (HWE now has its own file). The other filenames have been changed accordingly.  I also corrected my mistake and changed the protein fname to read from data.dir.
